async-nats/src/lib.rs

```diff
@@ -850,8 +870,6 @@ impl ConnectionHandler {
                 queue_group: None,
             });
         }
-
-        self.connector.events_tx.try_send(Event::Connected).ok();
```

Comment:
Who's sending this now?

Reply:
It's sent at the end of successful connect in `try_connect`.